doc/example/general.rst

```diff
@@ -74,7 +74,7 @@ which will make the sphinx tests part of your test run.
 Selecting one or more environments to run tests against
 --------------------------------------------------------
 
-Using the ``-e ENV[,ENV2,...]``  option you explicitly list
+Using the ``-e ENV[,ENV36,...]``  option you explicitly list
```

Comment:
this one looks like a mistake too

Reply:
this one I think is fine :+1:

Reply:
I don't or I wouldn't have pointed it out.  This is a listing of environments and yes while most of them are usually python environments this is documenting the command line.